packages/api/controllers/user.js

```diff
@@ -20,4 +20,34 @@ const getUserById = async (req, res) => {
   }
 };
 
-module.exports = { getUserById };
+const create = async (req, res) => {
+  const { email, username } = req.body;
+  if (!email) {
+    return res.status(400).send('Missing email');
+  }
+  if (!username) {
+    return res.status(400).send('Missing username');
+  }
```

Comment:
What led you to check these specifically instead of letting Sequelize validation catch it?

Reply:
specifically, I wasn't thinking... I'll update to sequelize error-handling